### DISCUSSION TITLE
在自动选择模式快捷键 怎么配置

### DISCUSSION THREAD

**Discussion by alaric8** [gdh1995/vimium-c#1114] [Q&A]
1. 在选中文本的情况下我按下 ctrl+c  可以结束选择并复制到剪切板

**Comment by gdh1995** [comment]
`enterVisualMode` 的模式？回车就能做到“复制并退出”了。

这个模式下不好直接改快捷键，只能用 `mapKey <xxx:v> <yyy>` 改映射关系来间接配置快捷键，参考 https://github.com/gdh1995/vimium-c/wiki/Use-in-another-keyboard-layout#per-mode-mapkey
